docs/en/samples/juicefs_setup.md

```diff
@@ -0,0 +1,132 @@
+# Steps to build Juicefs open source environment
+
+In order to quickly start Fluid + JuiceFS, you can quickly build an open source version of JuiceFS environment in Kubernetes by following the steps below. This environment is only used for functional verification, without any tuning, and is not recommended for production environments.
```

Comment:
In order to get started with Fluid + JuiceFS, you can quickly build community version of JuiceFS environment in Kubernetes by following the steps below.